docs/src/extend/custom-rules-tutorial.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Custom Rule Tutorial
+eleventyNavigation:
+    key: custom rule tutorial
+    parent: custom rules
+    title: Custom Rule Tutorial
+    order: 1
+---
+This tutorial covers how to create a custom rule for ESLint. You can create custom rules to validates if your code meets a certain expectation, and determine what to do if it does not meet that expectation.
```

Comment:
```suggestion
This tutorial covers how to create a custom rule for ESLint. You can create custom rules to validate if your code meets a certain expectation, and determine what to do if it does not meet that expectation.
```

Reply:
already fixed in docs/src/extend/custom-rule-plugin-tutorial.md